Resources/views/Form/ckeditor.html.twig

```diff
@@ -0,0 +1,7 @@
+{% for plugin_name, plugin in ckeditor_plugins %}
+	{{ ckeditor_plugin(plugin_name, plugin) }}
+{% endfor %}
+
+{{ ckeditor_widget(ckeditor_field_id, ckeditor_configuration, {
+	input_sync: true,
+}) }}
```

Comment:
Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.